internal/restapi/browse/handler.go

```diff
@@ -143,3 +149,99 @@ func (h *browseHttpService) getModuleMetadataHandler() http.Handler {
 		h.responseHandler.Write(rw, data, http.StatusOK)
 	})
 }
+
+// GetReleasesHandler will return a list of all releases published.
+func (h *browseHttpService) getReleasesHandler() http.Handler {
+	return http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
+		params := mux.Vars(r)
+		ageStr := params["age"]
```

Comment:
Use a query argument for age and instead of using ParseUint() use [ParseDuration](https://pkg.go.dev/time@go1.21.4#ParseDuration)
then convert the resulting value to seconds before.

If the age query argument is not set assume a value of 7 days.

Reply:
ParseUint is actually used here to convert the time from `string` to `Unint` because the field `MaxAgeSeconds` requires time to be `*uint64`.
We are getting age from Front End in seconds so It's not required to further use ParseDuration.
Will add the default age to 7 days (604800 seconds).

Reply:
@aflahaha I suggest `ParseDuration` because the age is a duration, using ParseDuration allows the user to specify the age as `7d`  rather than `604800` seconds.